parser/tests/multiple_import.tree

```diff
@@ -0,0 +1,17 @@
+NodeTypeFile
+  end-rune = 63
+  start-rune = 0
+  child-node =>
+    NodeTypeImport
+      end-rune = 13
+      import-source = FooBar
+      start-rune = 0
+    NodeTypeImport
+      end-rune = 27
+      import-source = BarBaz
+      start-rune = 14
+    NodeTypeImport
+      end-rune = 63
+      import-source = Something
+      import-subsource = SomethingElse
+      start-rune = 28
```

Comment:
Yeah, I actually don't know if using files like this for tests is more idiomatic than raw string literals instead of `.go` files. I personally wouldn't worry about it, but pygithub tests work like this and I think it fucking sucks from a contribution perspective it isn't necessarily clear how to add tests when you add this indirection.